src/test/java/seedu/address/logic/commands/AddCategoryCommandTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.stubs.ModelStub;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.category.UserDefinedCategory;
+
+public class AddCategoryCommandTest {
+    private UserDefinedCategory toAdd = new UserDefinedCategory("test", "test");
+    private UserDefinedCategory toAdd2 = new UserDefinedCategory("test2", "test2");
+    private ModelStub model = new ModelStub();
+
+    @Test
+    public void addCategoryCommandTest() throws CommandException {
+        requireNonNull(model);
+        model.addCategory(toAdd);
+        model.addCategory(toAdd2);
```

Comment:
This test case should run an assertion to ensure that the add returns a success message when added.

Reply:
###

Reply:
You are right, fixing it now.

Reply:
Fixed